WordPress/MediaEditor/Capabilities/Crop/MediaEditorCropZoomRotate.swift

```diff
@@ -71,3 +65,13 @@ class MediaEditorCropZoomRotate: MediaEditorCapability {
         }
     }
 }
+
+extension MediaEditorCropZoomRotate: TOCropViewControllerDelegate {
+    func cropViewController(_ cropViewController: TOCropViewController, didFinishCancelled cancelled: Bool) {
+        onCancel()
```

Comment:
I'm slightly confused by `onFinishEditing()` not being called here when `cancelled = false` but, again, if this is intentional, a quick explanation and marking this as "Resolved" will do for me.

Reply:
`onFinishEditing()` is called when the user made some change in the image and tap "Done".

In this case, the user is simply dismissing the crop view controller. That's why we call `onCancel()`.

In `AztecPostViewController.swift` there's only an `onFinishEditing` block, but we can also add an `onCancel`. We don't have it because if the user cancel we simply do nothing.

Does that make sense?